ExomeDepth/exomedepth_db.py

```diff
@@ -167,5 +184,18 @@ def call_print_all_samples(args):
     parser_delete_sample.add_argument('flowcell_id', nargs='+', help='flowcell barcode')
     parser_delete_sample.set_defaults(func=call_delete_sample_db)
 
+    """ Arguments fill database based on cohort sampels"""
+    parser_fill_database = subparser.add_parser(
+        'fill_database',
+        help='Fill database based on folder path and BAM files'
+    )
+    parser_fill_database.add_argument('path', help='path of folder to be included')
+    parser_fill_database.add_argument(
+        '--conflict_file',
+        default="conflicts.txt",
+        help='output file name containing conflicts that must be resolved manually (default = conflicts.txt)'
+    )
```

Comment:
https://docs.python.org/3/library/argparse.html#filetype-objects Might be nice to use :).

Reply:
Good idea.